test/command/suite/io_flush/recursive/dependent/data_column/remove.test

```diff
@@ -0,0 +1,12 @@
+table_create Users TABLE_HASH_KEY ShortText
+column_create Users name COLUMN_SCALAR ShortText
+
+#@collect-query-log true
+io_flush --recursive dependent
+#@collect-query-log false
```

Comment:
What is tested here?

Reply:
@kou I want to test before/after `column_remove`, how io_flush output is changed.